src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/ActionModalForm.tsx

```diff
@@ -76,6 +76,16 @@ class ActionModalForm extends Component<Props, State> {
       this.setState({ actionType: 'code' })
     }
 
+    this.prepareActions()
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.actions !== this.props.actions) {
```

Comment:
is it possible that props.actions gets undefined ? IOW should we add `&& this.props.actions` ?

Reply:
The prepareActions method already does a null check and provides an empty array if it's the case